.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build & Test
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-2019, ubuntu-18.04 ]
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
```

Comment:
Are we sure we want a wildcard the last version part? This makes the build non-deterministic.

Note this is not just patches (i.e. bug fixes). IIRC the SDK versioning scheme allows new runtimes to be added with an "100" increment to this version, whereas "1" increments indicate a patch to the SDK itself.

Reply:
Fixed the version for now